### DISCUSSION TITLE
Getting account name by account ID in ELK

### DISCUSSION THREAD

**Discussion by Padmashankari** [vectordotdev/vector#22059] [Q&A] [answered]
Would it be possible to get aws_account_name by account_id from logs and map it to a seperate field in ELK? we are using vector agent

**Comment by jszwedko** [comment, marked as answer]
Unfortunately, I don't think there is anything natively in Vector to do that. You could always use the `lua` transform though to execute custom Lua scripts.

**Reply by Padmashankari** [reply]
Okay, Thanks @jszwedko
